src/Containers/BookList.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import { connect } from 'react-redux';
+import Book from '../Components/Book';
+import { removeBook } from '../Actions/index';
+
+const BooksList = ({ books, removeBook }) => {
+  const handleRemoveBook = (book) => {
+    removeBook(book);
+  };
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th>ID</th>
+          <th>Title</th>
+          <th>Category</th>
+        </tr>
+      </thead>
+      <tbody>
+        {
+          books && books.length
+            ? books.map((val) => (
+              <Book handleRemoveBook={handleRemoveBook} key={val.id} book={val} />
+            ))
+            : (
+              <tr>
+                <td>No books!</td>
+              </tr>
+            )
+      }
+
+      </tbody>
+    </table>
```

Comment:
Kindly fix the empty line and indentation.

Hint: If on VSCode, highlight your entire code and right-click and click Format document.